proto/minder/v1/minder.proto

```diff
@@ -91,7 +91,13 @@ message Cursor {
     // cursor is the index to start from within the collection being
     // retrieved. It's an opaque payload specified and interpreted on
     // an per-rpc basis.
-    string cursor = 1;
+    string cursor = 1 [
+        (buf.validate.field).string = {
+            pattern: "^[[:word:]=]*$",
+            max_len: 200,
+        },
+        (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
```

Comment:
If you use `*` as the pattern, you don't actually need this, since it will match a zero-length string.  It's fine to leave it, though.

Reply:
Fixed it in https://github.com/mindersec/minder/pull/4895 👍